ruby/downgrades/qlpack.yml

```diff
@@ -0,0 +1,5 @@
+name: codeql/ruby-downgrades
+groups: ruby
+version: 0.0.8-dev
```

Comment:
I think the outcome of the Slack discussion we had yesterday was that we would omit the `version` field from these downgrade packs since they are not being released as stand-alone packs but rather being bundled with the CLI.